.travis.yml

```diff
@@ -242,6 +242,26 @@ matrix:
     - $TRAVIS_BUILD_DIR/ci/travis_script_integration.sh
     # TODO(ARROW-4764): Using system java and plasma built with conda doesn't work
     # - $TRAVIS_BUILD_DIR/ci/travis_script_plasma_java_client.sh
+  - name: "Flight integration w/ OpenJDK 8, conda-forge toolchain"
```

Comment:
I'm curious: what do we gain by doing this in a separate matrix entry?

Reply:
@pitrou my rationale is that this can be a big/long check due to grpc dependency and the use of integration tests. If we start adding multi-lang tests, like the ipc-json suite, than I think it's worth adding a seperated entry (for parallelism).

Reply:
But if the Flight integration entry is doing a superset of what the non-Flight integration entry does, we're not winning anything (and we're probably losing parallelism credits).

Reply:
I could change it to just run the Flight tests. We'd still pay the cost of setting up and building everything though.

Reply:
How long does it take to run all integration tests in a single job?

Reply:
28 minutes, vs 24 for non-Flight tests. Ok, so I'll re-consolidate them, it doesn't seem to be much overhead (for now).